config/software/chef.rb

```diff
@@ -20,9 +20,13 @@
 
 relative_path "chef"
 
+fips_enabled = (project.overrides[:fips] && project.overrides[:fips][:enabled]) || false
```

Comment:
Best solution we could come up with :)  I was thinking "oh look, phony targets from makefiles" the entire time.